src/dune_rules/ocamldep.mli

```diff
@@ -32,6 +32,8 @@ val read_deps_of :
   -> Module.t
   -> Module.t list Action_builder.t
 
+val parse_compilation_units : modules:Modules.t -> string list -> Module.t list
```

Comment:
is this needed?

Reply:
No. Removed in https://github.com/ocaml/dune/pull/7187/commits/955fab5f9bced5987da8265b16f4ca2bcafcfd03.